tests/unit/test_parameters/test_parameter_values.py

```diff
@@ -1029,3 +1030,23 @@ def test_exchange_current_density_plating(self):
             match="referring to the reaction at the surface of a lithium metal electrode",
         ):
             parameter_values.evaluate(param)
+
+    def test_contains_method(self):
+        """Test for __contains__ method to check the functionality of 'in' keyword"""
+        parameter_values = ParameterValues(
+            {"Negative particle radius [m]": 1e-6, "Positive particle radius [m]": 2e-6}
+        )
+        assert (
+            "Negative particle radius [m]" in parameter_values
+        ), "Key should be found in parameter_values"
+        assert (
+            "Invalid key" not in parameter_values
+        ), "Non-existent key should not be found"
+
+    def test_iter_method(self):
+        """Test for __iter__ method to check if we can iterate over keys"""
+        parameter_values = ParameterValues(
+            values={"Negative particle radius [m]": 1e-6}
+        )
+        pv = [i for i in parameter_values]
+        assert len(pv) == 5, "Should have 5 keys "
```

Comment:
```suggestion
        assert len(pv) == 5, "Should have 5 keys"
```

Sorry for the nitpick :)